### PR TITLE
Enable import of native msbuild projects into CMake-generated solution

### PR DESCRIPTION
**Pull request by lokitoth**
This enables an option `Drlclientlib_DOTNET_USE_MSPROJECT=On` to generate a CMake-based solution file which directly imports the .NET projects (rather than relying on a custom command to drive `dotnet build`), and enables the native debug experiences in Visual Studio.

Also adds a utility target `launch_vs` to start Visual Studio passing the CMake build environment on to it; in particular, it enables passing the `BinaryOutputBase` environment variable to ensure that the output path of the managed assemblies is in the same places as the native interop assemblies, working around the issue that native assemblies do not get properly copied when dependent managed projects are built.

 Usage example (from my  `.vscode/settings.json`):

```
{
    "cmake.configureArgs": [
        "-DCMAKE_TOOLCHAIN_FILE=C:/w/vcpkg/scripts/buildsystems/vcpkg.cmake",
        "-DrlclientlibBUILD_DOTNET=On",
        "-Drlclientlib_DOTNET_USE_MSPROJECT=On"
    ]
}
```

Alternatively, when using CMake on the command line:

```
cmake -DCMAKE_TOOLCHAIN_FILE=C:/w/vcpkg/scripts/buildsystems/vcpkg.cmake 
      -DrlclientlibBUILD_DOTNET=On 
      -Drlclientlib_DOTNET_USE_MSPROJECT=On 
      -B build -S . 
cmake --build build --target launch_vs
```

### CODE REVIEW

File: bindings/cs/rl.net.cli.test/RegressionTest.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+
+        [TestMethod]
+        public void Test_PPTDesigner()
+        {
+            
+        }
+
+*/
+
```
Comment (jackgerrits) [inline bindings/cs/rl.net.cli.test/RegressionTest.cs:11]: Should this be here?

File: bindings/cs/rl.net.cli.test/RegressionTest.cs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+using Microsoft.VisualStudio.TestTools.UnitTesting;
```
Comment (jackgerrits) [inline bindings/cs/rl.net.cli.test/RegressionTest.cs:12]: This file seems like it may have been added by accident?

**Review by jackgerrits — APPROVED** [review]
Looks good apart from it looks like an unrelated test file snuck in

File: bindings/cs/rl.net.cli.test/RegressionTest.cs (hunk 1 above)
Comment (lokitoth) [reply bindings/cs/rl.net.cli.test/RegressionTest.cs:11]: Nope, good catch. Also there was a superfluous reference to the corresponding test file in the project.

**Review by jackgerrits — APPROVED** [review]

**Pull request merged**